pkg/gadgets/audit-seccomp/tracer/bpf/Makefile

```
@@ -1,19 +1,6 @@
 .PHONY: all
-all: audit-seccomp.o audit-seccomp-with-filter.o
-
-PKG_DIR=../../../..
-
-# See Linux' Makefile: when ARCH is i386 or x86_64, then SRCARCH is x86
-# It needs to match tests in bpf/bpf_tracing.h
-SRCARCH ?= x86
-
-audit-seccomp.o: audit-seccomp.c
-	clang -Werror -I$(PKG_DIR) -target bpf -O2 -g -c -x c $< -o $@ \
-		-D__KERNEL__ -D__TARGET_ARCH_$(SRCARCH)
-
-audit-seccomp-with-filter.o: audit-seccomp.c
-	clang -Werror -I$(PKG_DIR) -target bpf -O2 -g -c -x c $< -o $@ \
-		-D__KERNEL__ -D__TARGET_ARCH_$(SRCARCH) -DWITH_FILTER=1
+all:
+	GO111MODULE=on CGO_ENABLED=1 GOOS=linux go generate ../
 
 clean:
-	rm -f audit-seccomp.o audit-seccomp-with-filter.o
+	rm -f ../audit-seccomp*
```

It would be safer with the suffixes in case we add more source files:
```suggestion
	rm -f ../audit-seccompr_bpfel.go ../audit-seccompr_bpfel.o
```